control/filehash.go

```diff
@@ -66,8 +66,8 @@ func (d *DebianFileHash) Validate() (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	if size := stat.Size(); size != int64(d.Size) {
-		return false, fmt.Errorf("Error! Size mismatch! %d != %d", size, d.Size)
+	if size := stat.Size(); size != d.Size {
```

Comment:
This line is the real money shot -- no more cast since we're using the same type size.